package/gluon-mesh-vpn-core/luasrc/usr/lib/lua/gluon/mesh-vpn/core.lua

```diff
@@ -0,0 +1,69 @@
+local util = require "gluon.util"
+
+local M = {}
+
+function M.get_interface()
+  return 'mesh-vpn'
+end
+
+function M.get_method(name)
+  return require("gluon.mesh-vpn.method." .. name)
```

Comment:
Maybe we should find another term here, as "method" is already used in fastd configuration (and also appears in site.conf for this reason).

Reply:
How about `vpn_proto`?

Reply:
Protocol is the correct identifier if you want to differentiate between fastd, tunneldigger and wireguard. They're different protocols.

Reply:
I'm thinking about adding L2TP support to fastd though, making it an alternative to tunneldigger...

I think a more general term would be something like "feature provider" or "function provider" - unfortunately this could get mixed up with service providers :/